Application.py

```diff
@@ -0,0 +1,73 @@
+import Add_User as A
```

Comment:
If we re-structure in 2 files it is just needed to do:
from utils import * 
or just: 
import utils